### PR TITLE
Does  composer need to update with latest version.

### PR DESCRIPTION
**Pull request by vrkansagara**
update to the latest zf2 version.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "phly/phly-blog": "dev-master",
         "phly/phly-contact": "dev-master",
         "php": ">=5.3.3",
-        "zendframework/zendframework": "2.2.*",
+        "zendframework/zendframework": "2.4.0",
```
Comment (samsonasik) [inline composer.json]: actually, use <code>~2.4</code>

**Comment by vrkansagara** [comment]
it still looks like older version.

**Comment by samsonasik** [comment]
yes, now i think you can update to ~2.5.0

**Comment by vrkansagara** [comment]
there is no buddy are improving the zf-web composer.json  so it is closing this pull.

**Pull request closed without merging**